app/api/v1/resources_on_behalf.rb

```diff
@@ -0,0 +1,48 @@
+require 'services/publish_on_behalf_interactor'
+
+module API
+  module V1
+    # Default options for all API endpoints and versions
+    module ResourcesOnBehalf
+      extend ActiveSupport::Concern
+
+      included do
+        desc 'Takes a resource and an organization id, signs the resource '\
+             'on behalf of an organization, and publishes a new envelope with '\
+             'that signed resource',
+             http_codes: [
+               { code: 201, message: 'Envelope created' },
+               { code: 200, message: 'Envelope updated' }
+             ]
+
+        params do
+          use :update_if_exists
+          use :skip_validation
+        end
+
+        post 'resources/organizations/:organization_id/documents' do
+          authenticate!
+
+          params[:envelope_community] = select_community
```

Comment:
if we are only using `params[:envelope_community]` to pass a param to the interactor, there is no need for this.
```
interactor = PublishOnBehalfInteractor.call(
           envelope_community: select_community,
           organization_id: params[:organization_id],
           current_user: current_user,
           raw_resource: request.body.read
        )
```